### PR TITLE
updating ONBOARDING and 3PR

### LINKED ISSUE #211: videos for 2nd & 3rd PR
Refer these video tutorials:

Second PR from Git Bash : https://www.youtube.com/watch?v=MUGhso65ub8&t=262s&ab_channel=SamriddhiJain

Third PR from VS Code : https://www.youtube.com/watch?v=DGKGgXnP8Po&ab_channel=SamriddhiJain

### LINKED ISSUE #226: Tutorial For Third Pull Request
How I made my Third PR Using Git CLI

Note that I have already forked the twindle project and cloned it to my local machine and I already have git installed, steps on getting all of this set up have already been discussed on other issues.

- Step 1, Navigate to local twindle project folder
img1

- Step 2, Sync fork with the twindle directory
img2
img3
img5

- Step 3, After making desired changes, add it to the staging area so its ready to be commited
img6

- Step 4, Check the status and make sure everything is fine
img7
img8

- Step 5, Commit changes
img9

- Step 6, Push changes to github
img10
img11

- Step 7, Go to the Github page of the fork, and click on pull request or compare changes
Untitled design

- Step 8, Click on create pull request
img12

- Step 9, Add a title and/or comment and create your pull request
Untitled design (2)

# Helpful Resources
- Issues #41  and #156 
- [Git-Guides](https://github.com/git-guides/)
- [Learn Github in 20 minutes by Colt Steele on YouTube](https://www.youtube.com/watch?v=nhNq2kIvi9s)
- [Dzone.com - How to resolve github merge conflicts](https://dzone.com/articles/how-to-resolve-github-merge-conflicts)
- [GitHub docs - Resolving a merge conflict on github](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github)

I hope this was helpful!

### PR DESCRIPTION
**Pull request by piratelicorne**
ONBOARDING - updating tutorial links for second and third Pull Requests, closes #211, closes #226, closes #305

### CODE REVIEW

**Review by madhusudanbabar — APPROVED** [review]

**Review by tusharkandpal — APPROVED** [review]
Nice job! 👍

File: docs/ONBOARDING.md [hunk 1]
```diff
@@ -41,15 +43,23 @@ Once you are part of the twindle team, please join us here to keep in touch.
 - If you are using terminal(Mac) or command prompt(windows), watch this step by step video:
 
 [![tutorial](https://img.youtube.com/vi/7I9StcZt5cI/0.jpg)](https://youtu.be/7I9StcZt5cI).
+<br><br>
```
Comment (UnevenCoder) [inline docs/ONBOARDING.md:46]: for future reference 2 enter is treated as br

File: docs/ONBOARDING.md [hunk 2]
```diff
@@ -41,15 +43,23 @@ Once you are part of the twindle team, please join us here to keep in touch.
...
 4. Do the third PR
+<br>
 
 **What is the third PR about?**
 - Make few changes in the file `TEAM.md` inside the `docs` directory, commit, through the CLI (command line interface) and raise your second Pull Request.  
 
 
 **How to do the third PR?**
 
+- Check this step by step [tutorial](https://github.com/twindle-co/twindle/issues/226)
```
Comment (UnevenCoder) [inline docs/ONBOARDING.md:62]: Instead of check just do
[Step by Step tutorial](url)

File: docs/TEAM.md [hunk 3]
```diff
@@ -34,6 +34,7 @@
 |[Rajesh](https://github.com/rkumar1904)|I'm a Lead frontend developer from Mumbai, India. Total 9+ years experience in front end technology. Currently working on ReactJS & Flutter. Initially, I started my career as a Programming Faculty.|
 |[Sunny](https://github.com/SunnyDev7)|I am Sunny from India and a engineering student. And i am wroking on MERN stack and also comfortable with html, css and vanilla js and am Here to learn, collaborate and help.|
 |[Akshay Sharma](https://github.com/Akshay2996)|I'm a Frontend Developer from Bangalore, India. Total 1+ years of experience in the industry. Contributing to Twindle to make it successful.|
+|[Melissa Huerta](https://github.com/piratelicorne)|Hi! I'm Systems Engineer from Perú, 3+ yrs of experience in IT Infrastructure, I have knowledge in C and Javascript. I recently started in Development as Flutter Developer, currently working on side projects 👩🏽‍💻 and blogging for #WomenWhoCodeSV |
```
Comment (UnevenCoder) [inline docs/TEAM.md:37]: > I recently started in Development

you mean coding or meant web development ?

**Review by UnevenCoder — CHANGES_REQUESTED** [review]
Added some comments :)

File: docs/TEAM.md (hunk 3 above)
Comment (piratelicorne) [reply docs/TEAM.md:37]: I meant both, cause I have experience in IT Infrastructure and Production department but not in IT Development one.

File: docs/ONBOARDING.md (hunk 1 above)
Comment (piratelicorne) [reply docs/ONBOARDING.md:46]: yeah, for some reason it wasn't showing too much space in the preview, so when I added 2, it worked.

File: docs/ONBOARDING.md (hunk 2 above)
Comment (piratelicorne) [reply docs/ONBOARDING.md:62]: ok.

Comment (piratelicorne) [reply docs/ONBOARDING.md:62]: corrected.

File: docs/ONBOARDING.md (hunk 1 above)
Comment (piratelicorne) [reply docs/ONBOARDING.md:46]: modified.

**Review by ad3rinto — APPROVED** [review]
Fab.

**Comment by arthurbuhl** [comment]
Here is another small trick for closing multiple issues.

Right now, you are using `closes #211 #226 #305`. But if you hover above the `closes` text, it says this PR will only close issue #211 but not other.

The trick to close multiple issues is to use the keywords for each issue separated by comma.

**Example**: Closes #211, closes #226, closes #305

<hr/>

_P.S._ This can be linked to docs: #167

**Comment by piratelicorne** [comment]
Oh thank you🙏🏼🙏🏼, i didnt know

**Comment by piratelicorne** [comment]
Please, let me know what is missing from here :)

**Pull request merged**